src/SignalR/clients/ts/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj

```diff
@@ -3,17 +3,16 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <IsTestProject>false</IsTestProject>
+    <IsTestProject>true</IsTestProject>
     <BrowserTestHostName Condition="'$(ContinuousIntegrationBuild)' == 'true'">sauce.local</BrowserTestHostName>
     <_TestSauceArgs>--verbose --no-color --configuration $(Configuration) --sauce-user "$(SauceUser)" --sauce-key "$(SauceKey)"</_TestSauceArgs>
     <_TestSauceArgs Condition="'$(BrowserTestHostName)' != ''">$(_TestSauceArgs) --use-hostname "$(BrowserTestHostName)"</_TestSauceArgs>
-    <NpmTestArgs Condition="'$(DailyTests)' != 'true'">run test:local --no-color --configuration $(Configuration)</NpmTestArgs>
+    <NpmTestArgs Condition="'$(DailyTests)' != 'true'">run test:inner --no-color --configuration $(Configuration)</NpmTestArgs>
```

Comment:
What's the difference between test:local and test:inner?

Reply:
I've changed "test:inner" to only build the *.ts test files for the functional tests and "test:local" will build signalr, signalr-protocols-msgpack, functional test ts files, and the FunctionalTestApp.csproj.